src/shared/sol-conffile.c

```diff
@@ -285,9 +302,20 @@ _get_json_include_paths(
             continue;
         }
         if (sol_json_token_str_eq(&key, include_fallback, strlen(include_fallback))) {
-            *include_fallbacks = strndup(value.start + 1, sol_json_token_get_size(&value) - 2);
-            if (!*include_fallbacks)
+            char *buff;
+            struct sol_str_slice *s;
+            sol_json_token_remove_quotes(&value);
+            buff = strndup(value.start, sol_json_token_get_size(&value));
```

Comment:
Couldn't you get rid of this `strndup` and just append `value` to `include_fallbacks`, using`sol_json_token_to_slice`?